### DISCUSSION TITLE
Not able to see those charts which is created by the another user, while editing the dashboard

### DISCUSSION THREAD

**Discussion by umesh11111** [apache/superset#18426] [Ideas]
### Expected results
Those charts should be availale while editing the dashboard so we can add those chart which is created by another user.
what you expected to happen.

### Actual results
Getting those charts which is created by my userid.
what actually happens.

#### How to reproduce the bug

1. Go to '...' Any dashboard
2. Click on '....' on edit dashboard, then click on your charts& filters
3. Scroll down to '....' try to see those charts which is created by another user.

### Environment

(please complete the following information):

- superset version: `superset version`
- python version: `python --version`
- node.js version: `node -v`

### Checklist
any help would be appreciated. Thanks
Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by zuzana-vej** [comment]
If the chart owner is someone else - you can't add that chart into the dashboard, this is by design.  Are you referring to this case (chart owner is someone else) or are you referring to a case where you are co-owner of the chart and still can't add the chart to the dashboard?

**Comment by umesh11111** [comment]
> If the chart owner is someone else - you can't add that chart into the dashboard, this is by design. Are you referring to this case (chart owner is someone else) or are you referring to a case where you are co-owner of the chart and still can't add the chart to the dashboard?

Thanks for your response,
I am talking about chart owner is someonelse,then we are not able to see that chart while editing the dashboard,
we want to make this as enhancement.
